eslint.config.mjs

```
@@ -12,7 +12,7 @@ export default antfu({
   rules: {
     'ts/no-redeclare': 'off',
     'ts/consistent-type-definitions': ['error', 'type'],
-    'no-console': ['warn'],
+    'no-console': process.env.NODE_ENV === 'production' ? ['error'] : ['warn'],
```

ESLint

[error] Unexpected use of process.env.

(node/no-process-env)